### PR TITLE
New design

### PR DESCRIPTION
**Pull request by cristineguadelupe**
CleanShot 2022-04-13 at 14 05 05@2x
CleanShot 2022-04-13 at 14 06 09@2x
CleanShot 2022-04-13 at 14 06 20@2x
CleanShot 2022-04-13 at 14 07 20@2x

### CODE REVIEW

**Comment by cristineguadelupe** [comment]
Maybe we should put the info-box outside the borders...

**Comment by josevalim** [comment]
Yes, the info box should definitely be outside.

File: lib/assets/chart_builder/main.css [hunk 1]
```diff
@@ -6,61 +6,33 @@
...
-  display: flex;
-  flex-direction: row;
-  align-items: flex-start;
-  align-content: stretch;
-  flex-wrap: nowrap;
-}
 
-.row > *:not(:first-child) {
-  margin-left: 8px;
+  --primary-100: #ecf0ff;
```
Comment (jonatanklosko) [inline lib/assets/chart_builder/main.css:15]: Just so it matches Livebook :)

```suggestion
  --blue-100: #ecf0ff;
```

**Review by jonatanklosko — APPROVED** [review]
Fantastic, just one comment and ship it!

**Pull request merged**